src/status_im/utils/config.cljs

```diff
@@ -158,29 +153,18 @@
                [id network])
              default-networks)))
 
-(def link-preview-enabled-site?
-  #{"youtube.com"
-    "youtu.be"
-    "our.status.im"
-    "github.com"
-    "giphy.com"
-    "gph.is"
-    "media.giphy.com"})
-
-(def default-relay-provider "https://relay.walletconnect.com")
-
 (def default-wallet-connect-metadata {:name "Status Wallet"
                                       :description "Status is a secure messaging app, crypto wallet, and Web3 browser built with state of the art technology."
                                       :url "#"
                                       :icons ["https://statusnetwork.com/img/press-kit-status-logo.svg"]})
 
 (def wallet-connect-project-id "87815d72a81d739d2a7ce15c2cfdefb3")
 
-(def new-ui-enabled? (atom false))
+;;TODO for development only should be removed in status 2.0
+(def new-ui-enabled? false)
```

Comment:
this should be true

Reply:
oh good test, i just tested before merge :)